packages/mobx-react/src/inject.ts

```diff
@@ -80,7 +81,7 @@ export function inject(
 ): <T extends IReactComponent<any>>(
     target: T
 ) => T & (T extends IReactComponent<infer P> ? IWrappedComponent<P> : never)
-export function inject<S, P, I, C>(
+export function inject<S extends IKeyValueMap = {}, P extends IKeyValueMap = {}, I extends IKeyValueMap = {}, C extends IKeyValueMap = {}>(
```

Comment:
Why not 
https://github.com/mobxjs/mobx/blob/51a1338fcb5afc5a9bae5fff4f906e5d6f27592e/packages/mobx-react/src/types/IValueMap.ts#L1
as used here
https://github.com/mobxjs/mobx/blob/51a1338fcb5afc5a9bae5fff4f906e5d6f27592e/packages/mobx-react/src/types/IStoresToProps.ts#L1-L7
?

Reply:
I don't know much about the internals of MobX, but using `IValueMap` from `mobx-react` rather than `IKeyValueMap` from `mobx` does seem more straight-forward to me, too.

Technically, they're identical, so both solutions should work. But the reason for the error is that the generic type arguments need to be compatible with those in `IStoresToProps`. And that interface happens to use `IValueMap`, not `IKeyValueMap`.

Reply:
Thanks @urugator @DanielSWolf 

Yes, it makes sense to use `IVavalueMap` and I've changed it. The issue is that I got failing tests on CI but I can't reproduce it locally. That's why I marked the PR as Draft. I'll get back to the issue once find time 👍